Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V20.h

```diff
@@ -60,10 +55,21 @@
 #define Y_STOP_PIN                          PC8
 #define Z_STOP_PIN                          PB1
 
+//
+// Z Probe
+//
+#ifndef Z_MIN_PROBE_PIN
+  #define Z_MIN_PROBE_PIN                   PA0
+#endif
+
 //
 // Filament runout
 //
-#define FIL_RUNOUT_PIN                      PB5
+#if NUM_SERVOS
+  #define FIL_RUNOUT_PIN                    PA0
```

Comment:
It seems strange that the presence of a Servo Probe / BLTouch should force `FIL_RUNOUT_PIN` to conflict with the default `Z_MIN_PROBE_PIN`.